examples/node-app-mssql/README.md

```diff
@@ -66,6 +67,150 @@ $ cp connections.sync-db.json.docker connections.sync-db.json
 ```
 
 Then run (in order).
+
+```bash
+$ docker-compose up mssql
+$ docker-compose up app
+```
+
+# Node MSSQL Example (JavaScript)
```

Comment:
Remove everything above this? Some of the items above this is duplicate.

Reply:
Removed the duplicates.